apiserver/facades/client/bundle/bundle_test.go

```diff
@@ -1066,6 +1066,55 @@ applications:
 	s.st.CheckCall(c, 0, "ExportPartial", s.st.GetExportConfig())
 }
 
+func (s *bundleSuite) TestExportBundleWithApplicationStorage(c *gc.C) {
+	s.st.model = description.NewModel(description.ModelArgs{Owner: names.NewUserTag("magic"),
+		Config: map[string]interface{}{
+			"name": "awesome",
+			"uuid": "some-uuid",
+		},
+		CloudRegion: "some-region"})
+
+	args := s.minimalApplicationArgs(description.IAAS)
+	// Add storage constraints to the app
+	args.StorageConstraints = map[string]description.StorageConstraintArgs{
+		"storage1": {
```

Comment:
Probably worth having two storages defined here just to ensure it serialized multiple correctly.